{"uuid": "fb3f9fd3-5c21-43a8-8c8b-21fd12dd63ea", "children": ["691f7007-ee04-493b-9a53-bbac0a88fecf", "9789571d-5e94-4325-b275-45c5f537772a", "dd88bbff-fd59-477e-b4b6-98bc66303dfe"], "befores": [{"name": "env", "status": "broken", "statusDetails": {"message": "NameError: name 'env_config' is not defined\n", "trace": "  File \"/Users/icsoc/.virtualenvs/pytest/lib/python3.6/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/icsoc/.virtualenvs/pytest/lib/python3.6/site-packages/_pytest/fixtures.py\", line 923, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/icsoc/.virtualenvs/pytest/lib/python3.6/site-packages/_pytest/fixtures.py\", line 782, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/icsoc/work/pytest/test/conftest.py\", line 21, in env\n    return env_config\n"}, "start": 1548922366061, "stop": 1548922366061}], "start": 1548922366061, "stop": 1548922366148}